Changelog.md

```diff
@@ -1,5 +1,47 @@
 ## Ignition Tools 1.x
 
+### Ignition Tools 1.1.0 (20XX-XX-XX)
+
+1. Use ; as IGN_CONFIG_PATH delimiter on windows
+    * [Pull request #45](https://github.com/ignitionrobotics/ign-tools/pull/45)
+
+1. Clean up ignition-tools.pc file
+    * [Pull request #40](https://github.com/ignitionrobotics/ign-tools/pull/40)
+
+1. Fix cmake warning and error mesage types
+    * [Pull request #41](https://github.com/ignitionrobotics/ign-tools/pull/41)
+
+1. Infrastructure updates
+    * [Pull request #42](https://github.com/ignitionrobotics/ign-tools/pull/42)
+    * [Pull request #38](https://github.com/ignitionrobotics/ign-tools/pull/38)
+    * [Pull request #37](https://github.com/ignitionrobotics/ign-tools/pull/37)
+    * [Pull request #29](https://github.com/ignitionrobotics/ign-tools/pull/29)
+    * [Pull request #22](https://github.com/ignitionrobotics/ign-tools/pull/22)
+    * [Pull request #17](https://github.com/ignitionrobotics/ign-tools/pull/17)
+    * [Pull request #11](https://github.com/ignitionrobotics/ign-tools/pull/11)
+    * [Pull request #add gitignore](https://github.com/ignitionrobotics/ign-tools/pull/add gitignore)
```

Comment:
something wrong with the changelog script here

Reply:
> something wrong with the changelog script here

Fixed in 887ac4e.